mlkem/src/native/riscv64/riscv64-notes.md

```diff
@@ -0,0 +1,8 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
```

Comment:
Minor: Rename file to `README.md`

Reply:
What do you think about removing this file entirely and adding the important info the our root README?